src/platform-apps/channels/attachment-cards/index.tsx

```diff
@@ -31,6 +38,7 @@ export default class AttachmentCards extends React.Component<Properties, undefin
             <AttachmentCard
               key={attachment.url}
               attachment={attachment}
+              onRemove={this.props.onRemove ? this.itemRemoved.bind(this, attachment) : null}
```

Comment:
We send the attachment into `AttachmentCard`, right? It should publish `onRemove` with that value so we don't have to bind things here.

Reply:
I tried without bind (`onRemove={this.props.onRemove ? this.itemRemoved : null}`) but it doesn't work 🤔

Reply:
Sorry, I wasn't clear in my first comment. What I mean is in the `AttachmentCard` component it should have an `onRemove` that calls the passed in handler with the actual thing:

`AttachmentCard`:
```
  onRemove = () => {
    this.props.onRemove(this.props.attachment);
  }
```

Now this spot no longer needs to bind because it'll just receive the attachment automatically from the AttachmentCard.

Reply:
done